### DISCUSSION TITLE
Data integration from multiple devices

### DISCUSSION THREAD

**Discussion by splGH** [PhenoApps/Field-Book#974] [Q&A] [answered]
Multiple people may use multiple devices to collect data during a growing season. What are the best practices for data integration and updating all devices prior to data recording?

**Comment by trife** [comment, marked as answer]
@splGH this is definitely still a challenge. The ideal situation is uploading your data to a database via BrAPI which then allows downstream devices to pull those updates. This isn't ideal since not everyone has this type of setup. We have aspirations to build more robust systems for in-field sharing (see #465), but need to work on 1) the parts of the app that are being funded and 2) the parts of the app that will have the most impact for the most people.

One solution that others have implemented is exporting/merging all of the databases and then reimporting them on each device. Not ideal or fast, but it is doable.
